src/js/fetching/fetch-movies.js

```diff
@@ -102,15 +99,17 @@ export const getMovieById = async movieId => {
       `Poniżej console.log dla jednego filmu (${data.original_title}) po movieId: ${movieId}`
     );
     console.log(data);
-    //TO DO function here!
+
+    //TODO function here!
     return data;
   } catch (error) {
     console.error(error);
   }
 };
 
-//  5.    ------ Function - fetch - get trailer's url by movie ID ------
-export const returnTrailerUrlByMovieId = async movieId => {
+//  5.    ------ Function fetch - get trailer's url by movie ID ------
```

Comment:
The comment must be on the line above the content being described (no break line)